using-kontena/nodes.md

```diff
@@ -95,3 +95,33 @@ The command that may be used for inspecting a Kontena Node's health status.
 ```
 $ kontena node health <NODE_ID>
 ```
+
+## Manage Kontena Node availability
```

Comment:
Node availability general text should go to `README.md`, only usage should be in this page.

Reply:
Moved.